test/Access_Control/Curio/README.md

```diff
@@ -0,0 +1,198 @@
+# Curio Privilege Escalation
+- **Type:** Exploit
+- **Network:** Ethereum 
+- **Total lost**: ~16MM USD 
+- **Category:** Access Control
+- **Vulnerable contracts:**
+- - [Curio Chief](https://etherscan.io/address/0x579A3244f38112b8AAbefcE0227555C9b6e7aaF0#code)
+- - [Curio Pause](https://etherscan.io/address/0x1e692eF9cF786Ed4534d5Ca11EdBa7709602c69f#code)
+
+- **Attack transactions:**
+- - [Privilege escalation and mint](https://etherscan.io/tx/0x4ff4028b03c3df468197358b99f5160e5709e7fce3884cc8ce818856d058e106)
+  
+- **Attacker Addresses**: 
+- - Attacker EOA: [0xdaAa6294C47b5743BDafe0613d1926eE27ae8cf5](https://etherscan.io/address/0xdaAa6294C47b5743BDafe0613d1926eE27ae8cf5)
+- - Attacker Contract (verified): [0x1E791527AEA32cDDBD7CeB7F04612DB536816545](https://etherscan.io/address/0x1E791527AEA32cDDBD7CeB7F04612DB536816545)
+  
+- **Attack Block:**:  `19,498,911`  
+- **Date:** Mar 23, 2024
+- **Reproduce:** `forge test --match-contract=Exploit_Curio -vvv`
+
+## Step-by-step 
+1. The attacker locked tokens into the `DSChief` contract.
+2. Accumulated enough voting power.
+3. Displaced (lifted) the current hat and set their contract as the new hat (without requiring many tokens).
+4. The `DSPause` contract, upon making a `delegatecall`, checked `DSChief.canCall`, which returned true for the attacker's contract.
+5. Consuming the authorized minter role from `DSPause`, the attacker executed a malicious instructions through a custom `Spell` contract to mint tokens directly to themselves.
+
+
+## Detailed Description
+Curio Finance leverages MakerDAO contracts for managing specific protocol functionalities. Their CSC Curio token, a `DSToken`, designates the `DSPauseProxy` contract as its minter. The `DSPause` contract relies on `DSChief.canCall` to verify if an authorized (privileged) call is permitted.
+Then, executes a `delegatecall` through its proxy.
+
+The operational vulnerability resided in the `DSChief` contract, which manages a HAT account that can be reconfigured with not much voting power. The attacker exploited this by locking tokens in `DSChief`, getting enough votes to lift and displace the current hat, subsequently positioning their contract as the new hat. The main issue was that the previous hat (Curio's `DSSSpell`) did not have enough voting power to consider the system safe, which allowed the easy and cheap displacement. 
```

Comment:
```which manages a HAT account that can be reconfigured with not much voting power``` 

_Can_ be reconfigured or _was configured_?

```The main issue was that the previous hat (Curio's `DSSSpell`) did not have enough voting power to consider the system safe```

Similarly, consider the system safe is what regard? If `DSSSpell` had enough voting power, what would have been prevented? 

I think this paragraph can benefit from removing the "cans" and other generally meaningless adjectives (eg: "safe") and replace them with facts.

Reply:
More: the paragraph starts with "the operational vulnerability resided in the `DSCHief`..." but then later "the main issue was that the previous hat". So which one is it?